src/llama_stack_client/lib/cli/llama_stack_client.py

```diff
@@ -34,14 +34,12 @@ def cli(ctx, endpoint: str, config: str | None):
     ctx.ensure_object(dict)
 
     # If no config provided, check default location
-    if config is None:
+    if config:
```

Comment:
Is this change correct?

Reply:
yes, if config is provided, only then we need to enter this condition.

Reply:
thanks for catching this. fixed.